.github/workflows/api-rspec.yml

```diff
@@ -0,0 +1,31 @@
+# Some notes on getting started https://dev.to/sulmanweb/github-actions-run-rspec-test-on-every-push-for-free-1pb6#settings-through-the-yaml-file
+name: CI RSpec Tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: CI
+    runs-on: ubuntu-latest
+    env:
+      RAILS_ENV: test
+      # RAILS_MASTER_KEY: 'This needs to be securely set for GitHub Actions'
```

Comment:
@Donnadieu This needs to be set from the contents of `config/credentials/test.key` as ENV vars for the repo with GitHub Actions